product_examples/errors-and-omissions/products/errors-omissions/policy/exposures/professional_indemnity/exposure.json

```diff
@@ -0,0 +1,237 @@
+{
+    "name": "professionalindemnity",
+    "displayName": "Professional Indemnity",
+    "fields": [
+      {
+        "name": "professionalindemnity_type",
+        "title": "Professional Indemnity Type",
+        "type": "select",
+        "width": 3,
+        "values": [
```

Comment:
This width should be bigger as well. 
![Screen Shot 2019-06-28 at 3 43 35 PM](https://user-images.githubusercontent.com/3634095/60374913-a5235100-99bb-11e9-9fc6-7404550d7171.png)